<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSong">
    <property name="title">
     <string>Song</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>Library</string>
    </property>
    <addaction name="actionEdit_directories"/>
    <addaction name="actionReload_all"/>
    <addaction name="separator"/>
    <addaction name="actionUsed_tags"/>
    <addaction name="actionUsed_file_formats"/>
    <addaction name="actionSet_default_tags_file_formats"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionrepaint"/>
   </widget>
   <addaction name="menuSong"/>
   <addaction name="menuLibrary"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit_directories">
   <property name="text">
    <string>Edit directories ...</string>
   </property>
  </action>
  <action name="actionReload_all">
   <property name="text">
    <string>Reload all</string>
   </property>
  </action>
  <action name="actionUsed_tags">
   <property name="text">
    <string>Used tags ...</string>
   </property>
  </action>
  <action name="actionUsed_file_formats">
   <property name="text">
    <string>Used file formats ...</string>
   </property>
  </action>
  <action name="actionSet_default_tags_file_formats">
   <property name="text">
    <string>Set default tags &amp; file formats</string>
   </property>
  </action>
  <action name="actionrepaint">
   <property name="text">
    <string>repaint</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
